### PR TITLE
feat(tick): use requestAnimationFrame instead of setInterval

### PR DESCRIPTION
**Pull request by arnaud-zg**
## Why use `requestAnimationFrame` ?
I read many articles, I think it's better to use `requestAnimationFrame` rather than `setInterval`. I made a little refactoring, I replace `setInterval` by `requestAnimationFrame` and `clearInterval` by `cancelAnimationFrame`.

Source
- https://developers.google.com/web/fundamentals/performance/rendering/optimize-javascript-execution#use_requestanimationframe_for_visual_changes

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -21,44 +21,61 @@ export default class OauthPopup extends React.PureComponent<props> {
...
-        onCode(code, params);
-        this.externalWindow.close();
-      } catch (e) { }
-    }, 20);
-
+    this.animationFrameId = requestAnimationFrame(this.tick);
     this.externalWindow.onbeforeunload = () => {
-      this.props.onClose();
-      clearInterval(this.codeCheck);
+      this.closeExternalWindow
```
Comment (Ramshackle-Jamathon) [inline src/index.js]: missing `()`?

Comment (arnaud-zg) [reply src/index.js]: Damn you are right, it's fixed now 💫

**Comment by kalicki** [comment]
@Ramshackle-Jamathon is it possible to add this update in the NPM?

**Comment by Ramshackle-Jamathon** [comment]
@kalicki published as `0.1.1`

**Pull request merged**